src/js/providers/tracks-mixin.js

```diff
@@ -385,6 +385,10 @@ define(['../utils/underscore',
         return providerName.indexOf('flash') === -1 && (utils.isChrome() || utils.isIOS() || utils.isSafari());
     }
 
+    function _kindSupported(kind) {
```

Comment:
@pajong Not sure all the kinds can be, but this is not exactly the same as `if (track.kind === 'subtitles' || track.kind === 'captions')`